fe1-web/src/features/events/functions/events.ts

```diff
@@ -23,8 +23,15 @@ export const categorizeEventsByTime = (time: Timestamp, events: EventState[]) =>
   const upcomingEvents: EventState[] = [];
 
   events.forEach((e: EventState) => {
-    // if end time is set, the event has ended
-    if (e.end) {
+    // ended events are either:
+    // - meetings that have ended
+    // - other events that have been marked as ended (e.end is set)
+    // TODO: should find a way to do this without relying on the event type
+    // for example, by making that events ends when organizers mark them as ended (same as the others)
+    if (
+      (e.eventType === 'MEETING' && e.end && e.end <= t) ||
+      (e.eventType !== 'MEETING' && e.end)
```

Comment:
why not just keep the `e.end && e.end <= t` logic ? 
In a way isn't it weird when a roll call or an election immediately jumps into "past events" the moment it's closed (i.e. the results are available) ?

Reply:
I guess it would make sense, I initially didn't want to do this, because it would change the general behavior we're used to regarding roll call or election.

Reply:
I agree with @pierluca. It is better if we have the same behaviour for all event types.